docs/source/compatibility.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Compatibility
+description: Compatibility
```

Comment:
Cut description (it appears directly below the title and is redundant in this case)